interview-questions.md

```diff
@@ -7,53 +7,53 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. Rails follows an MVC pattern. What does that mean?
 
-  Your answer:
+  Your answer:It is a model view controller, it lets you organize your code better
 
-  Researched answer:
+  Researched answer:MVC is short for Model View Controller and it organizes your code into their own, neatly organized boxes
 
 
 
 2. What is a relational database? Are there other kinds of databases?
 
-  Your answer:
+  Your answer:Relational database is used to find relations on the items in the database. Yes there are other types of databases
 
-  Researched answer:
+  Researched answer: A databse structured to recognize relations among stored items of information.
 
 
 
 3. What is object-relational mapping?
 
-  Your answer:
+  Your answer:Connects the objects to the tables
 
-  Researched answer:
+  Researched answer: ORM is the idea of being able to write queries using the object oriented paradigm of your preferred programming language
 
 
 
 4. What is a gem?
 
-  Your answer:
+  Your answer: A gem is a ruby package that contains built in methods and functions specific to that gem
 
-  Researched answer:
+  Researched answer: A packaged Ruby application, or library
 
 
 
 5. What are primary keys? Why are they important?
 
-  Your answer:
+  Your answer:Primary keys are identifiers of objects in a table by their postion. They are important because they let you select the item by its key
 
-  Researched answer:
+  Researched answer:Primary keys are the placement of each item in a table and they are important because you have a specfic index to reference.
 
```

Comment:
These are all clean concise answers that articulate exactly what the question asked. Great job! Since you have it down so well its also okay to take time expounding on the topic to show how much you've interacted with the idea. Thats the sort of thing interviewers are looking for, not BS, but experience.